### DISCUSSION TITLE
vega-embed and interactivity?

### DISCUSSION THREAD

**Discussion by c-cube** [vega/vega-lite#7945] [Q&A] [answered]
Hi, I've tried to use vega-embed to build a plot into a web page, and it works very well. However, there is an interactive component to it (see below) which seems to be totally ignored, despite it working in the vega-lite editor. Is that not supported in vega-embed?
```json
"params": [
        {"name": "grid", "select": {"type": "interval"}, "bind": "scales"}
],
```

**Comment by domoritz** [comment, marked as answer]
They should be functionally equivalent. Make sure you have the right versions and that there are no errors in the browser console.

**Reply by c-cube** [reply]
Strangely, it works when I change the cdn URL I was using (to `https://cdn.jsdelivr.net/npm/vega-lite@5.2/build/vega-lite.min.js` from `https://cdn.jsdelivr.net/npm/vega-lite@5`). I changed it to try and get source maps and now this works :thinking:

**Comment by marr** [comment]
Can you share an example of this? I'm trying to get an external input bound to the selection. Thanks!
